docker/qgisserver/tests/functional/accesscontrol_allow_to_edit_test.py

```diff
@@ -52,8 +56,12 @@ def test_data2(dbsession):
 
     user_no_access = User("user_no_access", roles=[role1])
     user_full_access = User("user_full_access", roles=[role2])
+    user_full_access.id = 2147483647  # root
```

Comment:
Here `user_full_access` seems not a root user, but a user with full access role.
Root user seems to be a role used for c2cgeoportal tests not for QGIS server plugin test.